test/unit/org/apache/cassandra/db/compaction/BaseCompactionStrategyTest.java

```diff
@@ -236,6 +237,12 @@ SSTableReader mockSSTable(int level,
         when(ret.getGeneration()).thenReturn(new SequenceBasedSSTableUniqueIdentifier(level));
         when(ret.toString()).thenReturn(String.format("Bytes on disk: %s, level %d, hotness %f, timestamp %d, first %s, last %s, disk index: %d, repaired: %b, pend. repair: %b",
                                                       FBUtilities.prettyPrintMemory(bytesOnDisk), level, hotness, timestamp, first, last, diskIndex, repaired, pendingRepair));
+        Path filenamePath = Mockito.mock(Path.class);
+        when(filenamePath.toString()).thenReturn("mocked_path-" + ret.hashCode());
+        Path path = Mockito.mock(Path.class);
+        when(path.getFileName()).thenReturn(filenamePath);
```

Comment:
I think I am missing something, but I don't understand if they are used anywhere and what it is the value to create these two mocks.

Reply:
Oh good catch, that seems to be something only relevant in DSE and not in CC; I will revert that.